src/vstest.console/TestPlatformHelpers/TestRequestManager.cs

```diff
@@ -539,17 +525,8 @@ private bool RunTests(IRequestData requestData, TestRunCriteria testRunCriteria,
                 catch (Exception ex)
                 {
                     EqtTrace.Error("TestRequestManager.RunTests: failed to run tests: {0}", ex);
-                    if (ex is TestPlatformException ||
-                        ex is SettingsException ||
-                        ex is InvalidOperationException)
-                    {
-                        ConsoleLogger.RaiseTestRunError(this.testRunResultAggregator, ex);
-                        success = false;
-                    }
-                    else
-                    {
-                        throw;
-                    }
+                    testRunResultAggregator.MarkTestRunFailed();
```

Comment:
As `testRunResultAggregator` required only for console scenarios. I think we can remove this?